packages/fluent-theme/src/fluent/styles/DetailsList.styles.ts

```diff
@@ -41,5 +47,41 @@ export const DetailsRowStyles = (props: IDetailsRowStyleProps): Partial<IDetails
         }
       ]
     ]
+    // TODO: check with designer if needed according to relines.
+    // checkCell: {
+    //   // Move the next cell after check to the left to align to redlines.
+    //   // Temporary using a hard coded value until fluent becomes default and we can access the value from the default constant.
+    //   marginRight: -12
+    // }
+  };
+};
+
+export const DetailsRowCheckStyles = (props: IDetailsRowCheckStyleProps): Partial<IDetailsRowCheckStyles> => {
+  return {
+    check: {
+      // Currently this causes a vertical scroll due to column width calculation logic
```

Comment:
Thank you for the detailed comments--these are really helpful for when these styles do eventually become the default in v7.

Reply:
Would it make sense to capture these comments as a diff which we can apply at a later date i.e. `7.x` release? This would remove the need for commented-out code while maintaining a reference in Github and a patch we can apply to adhere to new standards.

If we remove the related code in an isolated commit we can easily apply the inverse to a `7.x` branch.